src/components/home/Search.js

```diff
@@ -26,53 +25,43 @@ import {
 } from '../../config/selectors';
 import { SEARCH_RANGES } from '../../enums/searchRanges';
 
-const useStyles = makeStyles((theme) => ({
-  root: {
-    padding: theme.spacing(0.25, 0.5),
-    margin: theme.spacing(3, 'auto'),
-    display: 'flex',
-    alignItems: 'center',
-    width: '100%',
-  },
-  iconButton: {
-    padding: theme.spacing(1),
-  },
-  divider: {
-    height: theme.spacing(3),
-    margin: theme.spacing(0.5),
-  },
-  search: {
-    margin: theme.spacing(1),
-  },
-  searchOptions: {
-    marginLeft: theme.spacing(1),
-    marginBottom: theme.spacing(2),
-  },
-}));
+function Search({ handleClick, isLoading, range, handleRangeChange }) {
+  const [searchInput, setSearchInput] = useState(null);
 
-function Search({
-  handleSearch,
-  handleClick,
-  isLoading,
-  range,
-  handleRangeChange,
-}) {
   const { t } = useTranslation();
-  const classes = useStyles();
 
-  const handleKeyUp = (event) => {
+  const handleChange = (event) => {
+    setSearchInput(event.target.value.trim().toLowerCase());
+  };
+
+  const handleSearch = () => {
+    handleClick(searchInput);
+  };
+
+  const handleSearchOnClick = (event) => {
     if (event.keyCode === 13) {
```

Comment:
This is deprecated but the alternatives are not well documented anyway (+ compatibility issues with old browsers) https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode